tests/expectations/metrics/test_core.py

```diff
@@ -99,11 +110,71 @@ def test_mean_metric_pd():
         metrics_to_resolve=(desired_metric,), metrics=metrics
     )
     metrics.update(results)
-    assert results == {desired_metric.id: 2}
+    assert results == {desired_metric.id: result}
 
 
-def test_stdev_metric_pd():
-    engine = build_pandas_engine(pd.DataFrame({"a": [1, 2, 3, None]}))
+@pytest.mark.parametrize(
+    "dataframe,result",
+    [
+        [pd.DataFrame({"a": [1, 2, 3, None]}), 2],
+        [
+            pd.DataFrame({"a": [Decimal(2.0), Decimal(0.18781)]}),
```

Comment:
@TrangPham Same comment -- about trying a heterogenous column in addition to the above homogenous ones.  Thanks.

Reply:
@TrangPham On a second thought, columns with values that have different types (heterogeneous) are rare/unlikely.  So if this does not look meaningful after an initial quick experiments, let us abandon this and stick to homogeneous (same-type valued) columns (like you have it already).  Thank you.